ec2/spark_ec2.py

```diff
@@ -1007,6 +1023,11 @@ def real_main():
         print >> stderr, "ebs-vol-num cannot be greater than 8"
         sys.exit(1)
 
+    # Prevent breaking ami_prefix
```

Comment:
Sure, why not. Will wait for any other changes to this.
I think it's a pretty rare case, which currently would just fail at with "Could not resolve AMI at:  ...". But we may as well cover it.